### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by serlus**
Propose a separate layout of the forum and supporting material, such as the link commented on the video, the edited file link (which could be the specific commit) and some other specific material. So it could be inserted and updated separately from the forum.

### CODE REVIEW

File: pythonpro/modules/templates/topics/topic_detail.html [hunk 1]
```diff
@@ -75,7 +102,7 @@ <h1 class="mt-4 mb-3">{{ topic.title }}</h1>
                         }
 
                         function storeStats() {
-                            if (maxWatchedTime > 0) {
```
Comment (renzon) [inline pythonpro/modules/templates/topics/topic_detail.html:78]: This ab2 will cause an error on Javascript

File: pythonpro/modules/templates/topics/topic_detail.html [hunk 2]
```diff
@@ -43,7 +43,34 @@ <h1 class="mt-4 mb-3">{{ topic.title }}</h1>
...
+                                </ul>
+                                <div class="tab-content">
+                                    <div class="tab-pane fade in active" id="forum">
+                                        <p>
+                                            <div id='discourse-comments' class="mb-5"></div>
+                                        </p>
+                                    </div>
+                                    <div class="tab-pane fade" id="material_de_apoio">
+                                        <p>
+                                            Howdy, I'm in Section 2.
```
Comment (renzon) [inline pythonpro/modules/templates/topics/topic_detail.html:66]: I This this should not be here, probably resulting of code copy

File: pythonpro/modules/tests/test_topics_view.py [hunk 3]
```diff
@@ -123,6 +123,25 @@ def test_discourse_url(resp, topic):
...
+def test_tab_1(resp, topic):  # test para verificar se esta selecionada tab 1
+    dj_assert_contains(resp, 'href="#forum" data-toggle="tab">Fórum</a>')
+
+
+def test_tab_2(resp, topic):
+    dj_assert_contains(resp, 'href="#material_de_apoio" data-toggle="tab">Material de apoio</a>')
+
+
+def test_tab_content_forum(resp, topic):
+    """criar teste que verifique quando tab selecionada mostra
```
Comment (renzon) [inline pythonpro/modules/tests/test_topics_view.py:135]: Should be in English

**Comment by serlus** [comment]
> There are some issues and could you also squash all commits in only one?

I have been using this PR to learn more about the project and than go to other issues.

**Comment by renzon** [comment]
Hey Sérgio. I think support material is already present on each fórum post. But one thing would be awesome and people have already asked for it:  a place to write personal comments. E.g, where you can do personal annotation about the classe and it would be avalilable only for the user hearself. We would need to create a new table, Annotation, connection User and Topic, probably with a primary key composed by PK from User and Topic. You would learn a log with this.

**Comment by serlus** [comment]
Ok, I like this idea, 'challenge' accepted.

**Pull request closed without merging**